.github/workflows/linux.yml

```diff
@@ -131,7 +157,7 @@ jobs:
 
   # Build libamrex and all tutorials with DPC++
   tutorials-dpcpp:
-    name: DPCPP@PubBeta GFortran@7.5 C++14 [tutorials]
+    name: DPCPP@PubBeta GFortran@7.5 C++17 [tutorials]
```

Comment:
@WeiqunZhang please mark this as "not required" until the PR is merged, since I change the name of that test.